cloud/cloud_file_cache.cc

```diff
@@ -31,14 +43,13 @@ static void clear_callback_state() { callback_state.clear(); }
 // Touch the file so that is the the most-recent LRU item in cache.
 //
 void CloudEnvImpl::FileCacheAccess(const std::string& fname) {
-  if (!GetCloudEnvOptions().sst_file_cache) {
+  if (!cloud_env_options.hasSstFileCache()) {
```

Comment:
What happens if I start with non-zero cache and then set the cache size to zero? The code will assume that all files are local, but they are not. I think it will fail because `FileCacheAccess` will be a noop, but the file will not be present.

Reply:
FileCacheAccess is a just an LRU cache of the file metadata. In this case, FileCacheAccess will be a noop (like u said). But CloudEnvImpl.NewRandomAccessFile will still behave correctly because it will access either local or cloud versin of the sst file. @igorcanadi

Reply:
Okay, it will work correctly, but the behavior will be different than if we started with a cache size of zero - in one scenario we'll have all files locally, in the other some files will have to be loaded from the cloud. Let's merge this now to move faster, but something to think about.